addon/sidebar.js

```diff
@@ -2,6 +2,25 @@ function element(selector) {
   return document.querySelector(selector);
 }
 
+function applyDarkTheme() {
+  document.body.style.background = "#4a4a4f";
+  document.body.style.color = "#fff";
+  document.querySelector(".page").classList.add("dark-theme");
```

Comment:
looks like there's already a utility function written above on this page for this, we should use it.